deploy/helm_charts/envs/mixer_stanford.yaml

```diff
@@ -0,0 +1,25 @@
+# Helm config
+mixer:
+  gcpProjectID: datcom-stanford
+  serviceName: stanford.api.datacommons.org
+
+ingress:
+  name: mixer-ingress-standford
```

Comment:
Currently stanford instance uses multi-cluster ingress. Note that current templates don't create mcs, only normal ingress. If we use helm to update the instance then this yaml file will recreate a normal ingress, we'll have to switch over the DNS to point to the IP created by the new ingress.

https://pantheon.corp.google.com/kubernetes/discovery?mods=-monitoring_api_staging&project=datcom-stanford&pageState=(%22savedViews%22:(%22i%22:%224d1f1b7e59b9472f85fb2153d8c26221%22,%22c%22:%5B%5D,%22n%22:%5B%22website%22,%22default%22,%22gke-mcs%22%5D))

Reply:
For this PR, I am creating a deployment for the mixer (not the website). The ingress is in https://pantheon.corp.google.com/kubernetes/ingress/us-central1/mixer-us-central1/mixer/mixer-ingress-standford/details?mods=-monitoring_api_staging&project=datcom-stanford. Can you check if that is correct?

Reply:
sorry I was not aware that the stanford project have both the website an the mixer. Please disregard this comment and other related comments.